.github/workflows/linuxtests.yml

```diff
@@ -0,0 +1,58 @@
+name: Linux Tests
+
+on:
+  push:
+    branches:
+      - master
+      - maintenance/**
+  pull_request:
+    branches:
+      - master
+      - maintenance/**
+
+jobs:
+  Python_3.7_Debug
+    name: Python 3.7 Debug
```

Comment:
Can we have a description of what this entry does as a comment? Why is a build required against a debug version, what is it's importance in overall CI run? A future maintainer then finds it easier to decide whether to keep the entry or not (it may have just been an issue with an old Python version).

Reply:
That's been there for a very long time so I don't know but some folks run it occasionally.

Reply:
Can you do that? Generally, the debug builds are only available for default python, which is 3.6 on Ubuntu 18.04. NumPy uses 20.04 which has 3.8 as the default, and downloads the debug support packages with `apt get`.

Reply:
We have been doing that in Travis similarly. But what is "that" referring to in your question @charris ?